netbox/ipam/tests/test_api.py

```diff
@@ -604,6 +607,7 @@ def test_create_multiple_available_prefixes(self):
         self.assertHttpStatus(response, status.HTTP_201_CREATED)
         self.assertEqual(len(response.data), 4)
 
+
```

Comment:
This extra line is causing a PEP8, likewise could you clean up the other PEP8 errors?